### ISSUE TITLE
Installing on my Linux workstation - commands not found

### ISSUE THREAD

**Issue by Arxcis** [technocake/goto#3]
Here is the turn of events on my Linux box
```console
$ ~/git/technocake -> git clone git@github.com:technocake/goto.git
Cloning into 'goto'...
remote: Counting objects: 147, done.
remote: Compressing objects: 100% (14/14), done.
remote: Total 147 (delta 4), reused 16 (delta 4), pack-reused 128
Receiving objects: 100% (147/147), 193.94 KiB | 393.00 KiB/s, done.
Resolving deltas: 100% (75/75), done.

$ ~/git/technocake -> ls
goto

$ ~/git/technocake -> cd goto/

$ ~/git/technocake/goto on master -> ls
bin  gotomagic  install_goto.sh  README.md  requirements.txt  the_real_goto.py  win

$ ~/git/technocake/goto on master -> ./install_goto.sh 
ln: failed to create symbolic link '/usr/local/bin/*': Permission denied
./install_goto.sh: line 14: start_goto: No such file or directory
./install_goto.sh: line 17: project: command not found
./install_goto.sh: line 20: project: command not found
./install_goto.sh: line 23: goto: command not found
./install_goto.sh: line 24: goto: command not found
./install_goto.sh: line 25: goto: command not found

$ ~/git/technocake/goto on master -> sudo ./install_goto.sh 
[sudo] password for jonas: 

$ ~/git/technocake/goto on master -> sudo ./install_goto.sh 
[sudo] password for jonas: 
mkdir: cannot create directory ‘/home/jonas/.goto’: File exists
mkdir: cannot create directory ‘/home/jonas/.goto/projects’: File exists
./install_goto.sh: line 14: start_goto: No such file or directory
./install_goto.sh: line 17: project: command not found
./install_goto.sh: line 20: project: command not found
./install_goto.sh: line 23: goto: command not found
./install_goto.sh: line 24: goto: command not found
./install_goto.sh: line 25: goto: command not found

$ ~/git/technocake/goto on master -> goto list

Command 'goto' not found, did you mean:

  command 'gyoto' from deb gyoto-bin
  command 'wgoto' from deb wily
  command 'gotox' from deb dvb-apps
  command 'goo' from deb goo

Try: sudo apt install <deb name>

```

The contents of the `~/.goto` folder after the installation attempt
```console
$ ~/.goto -> find .
.
./active-project
./projects

$ ~/.goto -> cat active-project projects/
cat: projects/: Is a directory

```

**Comment by technocake** [comment]
Yes, good catch. The /usr/local/bin is a osx specific folder (and some other linux distroes) who is already included in the path. The strategy is to symlink into a bin folder that is already in the path. 

A solution might be to find another common folder that already are in path for most nix systems. Maybe even do a little os detection and do os specific steps during install.

**Comment by Arxcis** [comment]
On second glance, i think i did a mistake. 
I did
```
git clone git@github.com:technocake/goto.git
```
Instead I should have done
```
git clone git@github.com:technocake/goto.git /usr/local/opt/goto
```

**Comment by Arxcis** [comment]
Closing this as I have found my mistake. Re-opeing if it does not solve my issue.

**Comment by Arxcis** [comment]
Confirming that this fix worked for med on Ubuntu 18.04

> On second glance, i think i did a mistake.
> I did
> 
> ```
> git clone git@github.com:technocake/goto.git
> ```
> 
> Instead I should have done
> 
> ```
> git clone git@github.com:technocake/goto.git /usr/local/opt/goto
> ```
